### PR TITLE
apple-home-page-clone

### PR DESCRIPTION
**Pull request by tirthajyoti-ghosh**
This static page contains:
- Navbar built with CSS linear-gradient and CSS Sprite
- Main hero section with background image
- Promo section 
- Footer

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
@tirthajyoti-ghosh 
### Good job, this is impressive. However, there are a few changes you need to work on for your project to be approved.

- Make the banner background image bigger as in the [reference website](https://web.archive.org/web/20140301004610/http://www.apple.com/)
- Fix the spacing between the 4 thumbnails images, refer to [reference website](https://web.archive.org/web/20140301004610/http://www.apple.com/)
space

### once you make all these required changes, resubmit your project for review. All the best in your next submission.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+					</a>
+				</div>
+			</header>
+    </section>
+  </main>
+  
+  <div class="promo">
+    <ul>
+      <li>          
+        <a class="promo1" href="#">
```
Comment (euqueme) [inline index.html]: **.promo1** doesn't exist in your CSS file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" width="364" height="200" alt="iPhone 5s. Forward thinking.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo2" href="#">
```
Comment (euqueme) [inline index.html]: **.promo2** doesn't exist in your CSS file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" width="364" height="200" alt="iPhone 5c. For the colorful.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo3" href="#">
```
Comment (euqueme) [inline index.html]: **.promo3** doesn't exist in your CSS file.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg" width="364" height="200" alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo4" href="#">
```
Comment (euqueme) [inline index.html]: **.promo2** doesn't exist in your CSS file.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+  <div class="promo">
+    <ul>
+      <li>          
+        <a class="promo1" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" width="364" height="200" alt="iPhone 5s. Forward thinking.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo2" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" width="364" height="200" alt="iPhone 5c. For the colorful.">
```
Comment (euqueme) [inline index.html]: **width="364" height="200"**
Add the right selectors for width and height in your CSS file

File: index.html [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+				</div>
+			</header>
+    </section>
+  </main>
+  
+  <div class="promo">
+    <ul>
+      <li>          
+        <a class="promo1" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" width="364" height="200" alt="iPhone 5s. Forward thinking.">
```
Comment (euqueme) [inline index.html]: **width="364" height="200"**
Add the right selectors for width and height in your CSS file

File: index.html [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+        </a>    
+      </li>
+      <li>          
+        <a class="promo2" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" width="364" height="200" alt="iPhone 5c. For the colorful.">
+        </a>    
+      </li>
+      <li>          
+        <a class="promo3" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg" width="364" height="200" alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
```
Comment (euqueme) [inline index.html]: **width="364" height="200"**
Add the right selectors for width and height in your CSS file

File: index.html [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+        </a>    
+      </li>
+      <li>          
+        <a class="promo4" href="#">
+          <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" width="364" height="200" alt="Supplier Responsibility The latest results of our worldwide efforts.">
```
Comment (euqueme) [inline index.html]: **width="364" height="200"**
Add the right selectors for width and height in your CSS file

File: index.html [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+          <a href="#">Environment</a>
+        </li>
+        <li>
+          <a href="#">Job Opportunities</a>
+        </li>
+        <li>
+          <a href="#">Contact Us</a>
+        </li>
+        <li class="worldwide">
+          <a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/global/elements/flags/16x16/usa.png" alt="Choose your country or region" height="16" width="16"></a>
```
Comment (euqueme) [inline index.html]: **height="16" width="16"**
Add the right selectors for width and height in your CSS file

File: index.html [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+						<img class="small" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png" width="980" height="104" alt="What will your verse be?">
+					</a>
+				</h1>
+				<h2 class="subheadline">
+					<a href="#">
+						<img class="large" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_large.png" width="980" height="176" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+						<img class="medium" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_medium.png" width="980" height="86" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+						<img class="small" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
```
Comment (euqueme) [inline index.html]: Add the right selectors for width and height in your CSS file

File: index.html [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+					</a>
+				</h2>
+				<div class="center">
+					<a href="#">
+						<img class="medium" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_27pt.png" width="980" height="64" alt="Watch the “Your Verse” film and explore the stories">
+						<img class="large" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/cta_your_verse_46pt.png" width="980" height="112" alt="Watch the “Your Verse” film and explore the stories">
```
Comment (euqueme) [inline index.html]: Add the right selectors for width and height in your CSS file

File: index.html [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+  </header>
+  <main>
+    <section class="hero">
+      <a class="full" href="#">What will your verse be?</a>
+      <header class="header">
+				<h1 class="title">
+					<a href="#">
+						<img class="large" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_large.png" width="980" height="134" alt="What will your verse be?">
+						<img class="medium" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_medium.png" width="980" height="112" alt="What will your verse be?">
+						<img class="small" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small.png" width="980" height="104" alt="What will your verse be?">
```
Comment (euqueme) [inline index.html]: Add the right selectors for width and height in your CSS file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) :woman_technologist: 

### Hello @tirthajyoti-ghosh :wave: 

## Your page looks amazing :art: :eyes: and your Readme file is outstanding :muscle: but still there's a couple of mandatory request the code has to meet to be approved and that will make your project stand out even more :1st_place_medal: 

- [ ] Remove the inline styling from all of your **\** tags all the styling has to be defined in the CSS file.

- [ ]  Define in your CSS file or remove from your HTML file the promo1, promo2, promo3 and promo4 classes (you should only use classes defined in your CSS file).

### I have commented the lines of code that need to be fixed but its been a blast to review this project. :clinking_glasses: 

### Please :pray: correct these changes and submit back for a code review when you are done :rocket:

#### Happy Coding! :woman_technologist: 
STATUS OF THE PROJECT: REQUESTED CHANGES :traffic_light:

**Review by rOluochKe — APPROVED** [review]
@tirthajyoti-ghosh 
### This is amazing, you have done well. 

### I approve of the project, you can move to the next one. All the best

**Pull request merged**